draft-irtf-cfrg-vdaf.md

```diff
@@ -3995,6 +4016,8 @@ Aggregation involves simply adding up the output shares.
 ~~~
 def aggregate(Poplar1, agg_param, out_shares):
     (level, prefixes) = agg_param
+    if sum(prefix_counts) > len(out_shares):
```

Comment:
I don't think I understand what this check does. Where is `prefix_counts` defined?

Reply:
This was from an earlier version, removed it.